demos/TicTacToe/readme.md

```diff
@@ -0,0 +1,69 @@
+Firebase Cocos2d-x TicTacToe Sample
+==========================
+
+Desktop cocos2d-x sample for the Firebase C++ SDK.
+
+Introduction
+------------
+
+- [Read more about Firebase](https://firebase.google.com/docs)
+
+Prerequisites
+-------------
+- [Cmake](https://cmake.org/download/) (Minimum 3.6)
+
+- [Python 2.7](https://www.python.org/download/releases/2.7/) 
+
+- The latest version [of Cocos2d-x](https://cocos2d-x.org/download) (cocos environment variable must be added to PATH) 
+
+-  [Firebase CPP SDK](https://cocos2d-x.org/download) (Must be add to path under FIREBASE_CPP_SDK_DIR) 
+
+-  Windows: ([Visual Studio 2019](https://visualstudio.microsoft.com/downloads/))
+
+Getting Started
+---------------
+
+- Clone the firebase cpp quickstart GitHub repo.
+  ```
+  git clone https://github.com/firebase/quickstart-cpp.git
+  ```
+
+### Windows
+- Follow the steps in
+  [Set up your app in Firebase console](https://firebase.google.com/docs/cpp/setup?platform=android#desktop-workflow).
+
+- Place the google-services.json you just acquired in the demos/TicTacToe/google_services/ directory.
+
+- Navigate to the directory that you just cloned, step into demos/TicTacToe/ and run the setup script.
+  ```
+  python setup_demo.py 
+  ```
+
+- To debug with Visual Studio 2019, copy the google-services.json file into the tic_tac_toe_demo/build/ directory. Then open the .sln in that same directory in Visual Studio.
+
```

Comment:
A section describing how you play the game against another user, even if it's super simple, and the usages your demonstrating of  Database to store moves and win/los records, and how you use auth to sign in, would be grat.

Reply:
I really like this idea. I'm going to include a simple walk through on how to run through the game. Down the road I will do a deeper dive into each Firebase feature & the lines of code responsible for the specific Firebase actions with screenshots.

Reply:
How to play section added.